spec/support/.rubocop.yml

```diff
@@ -0,0 +1,2 @@
+inherit_from:
+  - ../../.rubocop.yml
```

Comment:
I suppose it seems weird, but I've been using technique in my projects, so it seems normal to me.

I don't think Rubocop is meant to tie you down to any particular standard, but just to help you enforce consistency.  That is way they provide so many ways to configure your project's standards to your liking.  We are being consistent in using the same rules for all Ruby files and the same rules to all RSpec files.

By the way, I found https://github.com/backus/rubocop-rspec in testing options.  It seems useful, but I would like to test it out on my projects before adding it here.